### ISSUE TITLE
Find a better way to display companies

### ISSUE THREAD

**Issue by jackiehluo** [coreatcu/website#5]
Right now, the logos for companies aren't that visually appealing... would be great if we could make them look uniform!

**Comment by anshulkgupta** [comment]
I've compressed them a little but I see what you mean. I think the problem is that most are rectangular and some are squares. Anyhow I think we should move all on a new page and link maybe 5-10 on the home page.

**Comment by jackiehluo** [comment]
Yeah, that's awkward. Maybe if we have a darker background? Technically, all of them are formatted as squares, but it's just hard to tell because some have a white background and two don't.

And that might be a good idea! How many companies do we want to include? I was thinking some of the main ones, but it's hard to decide which...

**Comment by anshulkgupta** [comment]
I've given this a pretty good overhaul. I think we can still throw in a scroll-spy there though

**Comment by jackiehluo** [comment]
Wait, are the companies still on the website? I don't see them!

**Comment by anshulkgupta** [comment]
working on bringing them back!
